src/components/core/tabs/tabs.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactNode, useEffect, useRef, useState } from 'react';
+import Tab from './tab';
+import css from './tabs.module.scss';
+
+export interface TabsProps {
+    active_index?: number;
+    contained?: boolean;
+    dark?: boolean;
+    children?: ReactNode[];
+}
```

Comment:
```suggestion
export interface TabsProps {
    active_index?: number;
    contained?: boolean;
    dark?: boolean;
    children?: ReactNode[];
}
```

Same case as the above, extend the DivElement attributes. Users may provide some overrides or their own custom event listener. Lets make it flexible in the design of our components